upgrade/1.2/scripts/upgrade/prerequisites.sh

```diff
@@ -240,7 +240,7 @@ if [[ $state_recorded == "0" ]]; then
         [[ -n ${USERNAME} ]]
 
         # Install our pit-init RPM and pull in any dependencies it has.
-        zypper --no-gpg-checks in -y pit-init
+        zypper --no-gpg-checks --plus-repo=https://packages.local/repository/csm-sle-15sp2 in -y pit-init
         /root/bin/bios-baseline.sh -y
```

Comment:
We will have a similar issue if bios-baseline.sh hits a failure.   I think we want a `set +e/-e` wrapped around this section so the err_report trap will catch possible failures.